src/Filters/filt_order.jl

```diff
@@ -430,31 +430,56 @@ lowest estimated filter order, with `ωn` specifying the cutoff or "-3 dB" frequ
 order and natural frequencies for an analog filter. The default domain is `:z`, interpretting the input 
 frequencies as normalized from 0 to 1, where 1 corresponds to π radians/sample.
 """
-function cheb2ord(Wp::Tuple{Real, Real}, Ws::Tuple{Real, Real}, Rp::Real, Rs::Real; domain::Symbol=:z)
+function cheb2ord(Wp::Tuple{Real,Real}, Ws::Tuple{Real,Real}, Rp::Real, Rs::Real; domain::Symbol=:z)
     Wps = (Wp[1] > Wp[2]) ? tuple(Wp[2], Wp[1]) : tuple(Wp[1], Wp[2])
     Wss = (Ws[1] > Ws[2]) ? tuple(Ws[2], Ws[1]) : tuple(Ws[1], Ws[2])
     (Wps[1] < Wss[1]) != (Wps[2] > Wss[2]) && throw(ArgumentError("Pass and stopband edges must be ordered for Bandpass/Bandstop filters."))
     ftype = (Wps[1] < Wss[1]) ? Bandstop : Bandpass
-    (Ωp, Ωs) = (domain == :z) ? (tan.(π/2 .* Wps), tan.(π/2 .* Wss)) : (Wps, Wss)
+    (Ωp, Ωs) = (domain == :z) ? (tan.(π / 2 .* Wps), tan.(π / 2 .* Wss)) : (Wps, Wss)
     if (ftype == Bandpass)
-        Wa = (Ωs.^2 .- (Ωp[1]*Ωp[2])) ./ (Ωs .* (Ωp[1]-Ωp[2]))
+        Wa = (Ωs .^ 2 .- (Ωp[1] * Ωp[2])) ./ (Ωs .* (Ωp[1] - Ωp[2]))
     else
         (Wa, Ωpadj) = chebyshev_bsfmin(Ωp, Ωs, Rp, Rs)
     end
     N = ceil(Int, chebyshev_order_estimate(Rp, Rs, minimum(abs.(Wa))))
 
     # new frequency for stopband spec.
     wnew = 1 / cosh(1 / N * acosh(√(db2pow(Rs) - 1) / √(db2pow(Rp) - 1)))
-    
+
     # lowpass prototype to analog filter re-map.
     Wna = zeros(2)
     if (ftype == Bandpass)
-        Wna[1] = (Ωp[1]-Ωp[2])/(2*wnew) + √( (Ωp[2]-Ωp[1])^2 / (4*wnew^2) + Ωp[1]*Ωp[2])
-        Wna[2] = (Ωp[1]*Ωp[2]) / Wna[1]
+        Wna[1] = (Ωp[1] - Ωp[2]) / (2 * wnew) + √((Ωp[2] - Ωp[1])^2 / (4 * wnew^2) + Ωp[1] * Ωp[2])
+        Wna[2] = (Ωp[1] * Ωp[2]) / Wna[1]
     else
-        Wna[1] = ((Ωpadj[1]-Ωpadj[2])*wnew)/2 + √((Ωpadj[1]-Ωpadj[2])^2 * wnew^2 / 4 + (Ωpadj[1]*Ωpadj[2]))
-        Wna[2] = (Ωpadj[1]*Ωpadj[2]) / Wna[1]
+        Wna[1] = ((Ωpadj[1] - Ωpadj[2]) * wnew) / 2 + √((Ωpadj[1] - Ωpadj[2])^2 * wnew^2 / 4 + (Ωpadj[1] * Ωpadj[2]))
+        Wna[2] = (Ωpadj[1] * Ωpadj[2]) / Wna[1]
     end
-    ωn = (domain == :z) ?  tuple((2/π)*atan(Wna[1]), (2/π)*atan(Wna[2]))  : tuple(Wna[1], Wna[2])
+    ωn = (domain == :z) ? tuple((2 / π) * atan(Wna[1]), (2 / π) * atan(Wna[2])) : tuple(Wna[1], Wna[2])
     N, ωn
+end
+
+"""
+    N = remezord(Wp, Ws, Rp, Rs)
+
+Order estimation for lowpass digital filter cases based
+on the equations and coefficients in [^Rabiner]. The original
+equation returned the minimum filterlength, whereas this implementation 
+returns the order (N=L-1). Results are valid within 1.3% error. `Wp` and 
```

Comment:
It's a bit unfortunate that this returns the order, but `remez` then takes the length. But I think `remez` is the one being inconsistent with taking `numtaps` instead of filter order. (Matlab's `remez` also takes the order.) But that ship has sailed for the moment...

But given that hardly anyone will look up the equation, I guess we can remove that sentence and replace it with a note warning about `remez` needing `L=N+1`.

Reply:
If we're looking to be consistent, we could keep this in terms of `numtaps` as remez requires! I'm not tied to the current commit, I think having a consistent API is more important. Whichever direction we decide on, I think you make a good point about noting the order vs. length values in the function header.

Reply:
Given that the IIR filter design functions all take order (what else should they), I think we should stay with order here, too, and live with `remez` being the odd one out for the moment.

Going forward, maybe we can change `remez` to take (mutually exclusive) keyword arguments `order` and `numtaps` and deprecate the positional `numtaps` argument. But that's clearly not in the scope of this PR.